rotkehlchen/api/v1/resources.py

```diff
@@ -2135,6 +2099,37 @@ def get(self, async_query: bool) -> Response:
         return self.rest_api.get_liquity_stats(async_query=async_query)
 
 
+class ModuleStatsResource(BaseMethodView):
+
+    get_schema = ModuleHistoryProcessingSchema()
+
+    @require_loggedin_user()
+    @use_kwargs(get_schema, location='json_and_query_and_view_args')
+    def get(
+            self,
+            module: ModuleWithStats,
+            async_query: bool,
+            from_timestamp: Timestamp,
+            to_timestamp: Timestamp,
+    ) -> Response:
+        if module in (ModuleWithStats.AAVE, ModuleWithStats.COMPOUND):
+            return self.rest_api.get_module_stats_using_balances(
+                async_query=async_query,
+                module=module.serialize(),
+                from_timestamp=from_timestamp,
+                to_timestamp=to_timestamp,
+            )
+
+        if module == ModuleWithStats.LIQUITY:
```

Comment:
since liquity is an option here shouldn't the old liquity stats endpoint be removed?